### DISCUSSION TITLE
Store object for Java client

### DISCUSSION THREAD

**Discussion by polperez** [nats-io/nats.java#707] [Ideas]
Hello and congratulations to the contributors. 

We develop our applications in Java and really appreciate the Java NATS client. 
Recently, we read about the new NATS feature called "object store". We would be interested in using it with our Java application. As far as I know, this feature is only available with Go (please tell me if I am wrong). 
Is there any plan to implement this feature in the Java client?
Thank you for your answer

Paul

**Comment by scottf** [comment]
Yes. In fact it's in progress, I hope to have it done by the end of the month. https://github.com/nats-io/nats.java/pull/705
